src/overlays/actors/ovl_Bg_Gjyo_Bridge/z_bg_gjyo_bridge.c

```diff
@@ -21,17 +29,94 @@ const ActorInit Bg_Gjyo_Bridge_InitVars = {
     (ActorFunc)BgGjyoBridge_Update,
     (ActorFunc)BgGjyoBridge_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Gjyo_Bridge/BgGjyoBridge_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Gjyo_Bridge/BgGjyoBridge_Destroy.s")
+InitChainEntry initChain[] = { ICHAIN_F32(unk_F8, 800, ICHAIN_CONTINUE),
+                               ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP) };
```

Comment:
```suggestion
InitChainEntry initChain[] = {
    ICHAIN_F32(unk_F8, 800, ICHAIN_CONTINUE),
    ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
};
```
I think clang might have done this because you didnt have a trailing comma on the last item in the list, not sure though

Reply:
Yeah that's why trailing commas are needed when you want proper vertical formatting.